packages/base-styles/_mixins.scss

```diff
@@ -1,6 +1,73 @@
 @import "./functions";
 @import "./long-content-fade";
 
+/**
+ * Typography
+ */
+
+@mixin heading-small() {
+	font-family: $font-family-headings;
+	font-weight: $font-weight-semi-bold;
+	font-size: $font-size-x-small;
+	line-height: $line-height-x-small;
+}
+
+@mixin heading-medium() {
+	font-family: $font-family-headings;
+	font-weight: $font-weight-semi-bold;
+	font-size: $font-size-medium;
+	line-height: $line-height-small;
+}
+
+@mixin heading-large() {
+	font-family: $font-family-headings;
+	font-weight: $font-weight-semi-bold;
+	font-size: $font-size-large;
+	line-height: $line-height-small;
+}
+
+@mixin heading-x-large() {
+	font-family: $font-family-headings;
+	font-weight: $font-weight-semi-bold;
+	font-size: $font-size-x-large;
+	line-height: $line-height-medium;
+}
+
+@mixin heading-2x-large() {
+	font-family: $font-family-headings;
+	font-weight: $font-weight-semi-bold;
+	font-size: $font-size-2x-large;
+	line-height: $font-line-height-2x-large;
+}
+
+@mixin body-small() {
+	font-family: $font-family-body;
+	font-weight: $font-weight-regular;
```

Comment:
This bit:

```
	font-family: $font-family-body;
	font-weight: $font-weight-regular;
```

is repeated across all body mixins.

Perhaps it can be its own mixin which is reused in each of those mixins?

Reply:
Sure, I'll try it.